pkgs/top-level/all-packages.nix

```diff
@@ -29916,7 +29916,7 @@ with pkgs;
   inherit (callPackage ../applications/virtualization/docker {})
     docker_24 docker_25 docker_26 docker_27;
```

Comment:
We should at least mark docker_24 as insecure.

Reply:
Good point. I'm back at a computer on Monday. Anybody with availability feel free to take over if you like.